src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -396,15 +399,22 @@ const CreateEditChannelView: React.FC<CreateEditChannelViewProps> = ({ newChanne
                 )}
               />
             </FormField>
-            <ActionBarTransaction
-              fee={FEE}
-              isActive={newChannel || (!loading && isDirty)}
-              fullWidth={!channelId}
-              primaryButtonText={newChannel ? 'Create channel' : 'Publish changes'}
-              secondaryButtonText="Cancel"
-              onCancelClick={() => reset()}
-              onConfirmClick={handleSubmit}
-            />
+            <CSSTransition
+              in={sheetState !== 'open'}
+              timeout={2 * parseInt(transitions.timings.loading)}
+              classNames={'actionbar'}
```

Comment:
Is this different than the regular `fade`?

Reply:
It has `transitions.timings.regular` delay on enter so it doesn't fade in immediately when the sheet begins to slide down